<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleWeights</class>
 <widget class="QWidget" name="VehicleWeights">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="rightCar">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>201</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Jeep1-01.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="leftCar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>201</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Jeep1-01.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="rearLeftWeight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rearRightWeight">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frontLeftWeight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frontRightWeight">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overallWeight">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="frontAxleWeight">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLabel" name="overallWeightLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Overall Weight</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rearAxleWeight">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>WT</string>
   </property>
  </widget>
  <widget class="QLabel" name="frontAxleLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Front Axle</string>
   </property>
  </widget>
  <widget class="QLabel" name="rearAxleLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rear Axle</string>
   </property>
  </widget>
  <widget class="Line" name="frontAxleLine">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="rearAxleLine">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLatLeft">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Length</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="comHt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Length</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLong">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Length</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="comLatRight">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Length</string>
   </property>
  </widget>
  <widget class="QLabel" name="comLongLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CoM Long</string>
   </property>
  </widget>
  <widget class="QLabel" name="comLatLeftLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CoM Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="comHtLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>CoM Ht</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="comLatRightLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CoM Width</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>321</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="heightCenterFrontWheelsLabel">
        <property name="text">
         <string>Ht of Cntr of Frt Wheels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="heightCenterRearWheelsRaisedLabel">
        <property name="text">
         <string>Ht of Cntr of R Wheels (raised)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="weightFrontWheelsRearRaisedLabel">
        <property name="text">
         <string>Weight on Frt Wheels (R raised)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="heightCenterFrontWheels"/>
      </item>
      <item>
       <widget class="QLineEdit" name="heightCenterRearWheelsRaised"/>
      </item>
      <item>
       <widget class="QLineEdit" name="weightFrontWheelsRearRaised"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
